scripts/bracket_orders.js

```diff
@@ -43,12 +49,7 @@ const argv = require("yargs")
     type: "int",
     describe: "Maximum auction batch for which these orders are valid",
     default: 2 ** 32 - 1,
-  })
-  .demand(["targetToken", "stableToken", "currentPrice", "masterSafe", "brackets"])
```

Comment:
Wasn't this already doing what this PR claims to do?

Reply:
Nope, this line makes the script fail if one of the required arguments is not there. The main goal of the PR is to fail if any unknown argument is used (so, .`strict()`).
Since I was at it I also moved the `demand` flag inside the definition of the required options for readability.

Reply:
Awesome!